flash/core/data/input_transform.py

```diff
@@ -406,17 +284,755 @@ def _create_collate_input_transform_processors(self) -> Tuple[Any]:
         worker_input_transform_processor = _InputTransformProcessor(
             self,
             worker_collate_fn,
-            getattr(self, func_names["per_sample_transform"]),
-            getattr(self, func_names["per_batch_transform"]),
+            self._per_sample_transform,
+            self._per_batch_transform,
             self.running_stage,
         )
         device_input_transform_processor = _InputTransformProcessor(
             self,
             device_collate_fn,
-            getattr(self, func_names["per_sample_transform_on_device"]),
-            getattr(self, func_names["per_batch_transform_on_device"]),
+            self._per_sample_transform_on_device,
+            self._per_batch_transform_on_device,
             self.running_stage,
             apply_per_sample_transform=device_collate_fn != self._identity,
             on_device=True,
         )
         return worker_input_transform_processor, device_input_transform_processor
+
+    ########################################
+    # HOOKS CALLED INTERNALLY WITHIN FLASH #
```

Comment:
nit: if they are internal, should they be at the bottom of the file?

Reply:
Done !